src/index.html

```diff
@@ -11,7 +11,134 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action ="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post">
+
+      <fieldset class="fieldset">
+
+        <legend>Personal information</legend>
+
+        <label class="label">
+          Surname:
+          <input type ="text" name ="Surname"
+           autocomplete="off">
```

Comment:
fix indentation everywhere

```suggestion
          <input
            type ="text"
            name ="surname"
            autocomplete="off"
         >
```